apps/gitness/src/pages-v2/user-management/user-management-container.tsx

```diff
@@ -131,18 +94,13 @@ export const UserManagementPageContainer = () => {
     {},
     {
       onSuccess: () => {
-        setCreateUserDialogOpen(false)
-        setResetPasswordDialogOpen(true)
         queryClient.invalidateQueries({ queryKey: ['adminListUsers'] })
-      },
-      onError: error => {
-        console.error(error)
       }
     }
   )
 
-  const handleCreateUser = (data: { uid: string; email: string; display_name: string }) => {
-    createUser({
+  const handleCreateUser: IDataHandlers['handleCreateUser'] = data => {
+    return promisifyMutation(createUser, {
```

Comment:
Instead of wrapping things in `promisifyMutation`, you can just the `mutateAsync` instead of `mutate` to get the promise-based version from Tanstack Query

https://tanstack.com/query/latest/docs/framework/react/guides/mutations#promises